src/utilities/schemaHandler.test.js

```diff
@@ -196,3 +235,13 @@ test('Checks preProcessSchema does not modify simple sample schema', () => {
   const processedSchema = preProcessSchema(testSchema4)
   expect(processedSchema).toEqual(testSchema4)
 })
+
+test('Checks if anyOf string/number are changed to decimal', () => {
+  const processedSchema = preProcessSchema(testSchema5)
+  expect(processedSchema.properties.test.anyOf).toStrictEqual([{ title: 'decimal', type: 'string', pattern: '^\\d+(\\.\\d{1,2})?$' }])
```

Comment:
Tests are failing because this needs to be updated to be the same as the one in schemaHandlers.js:
```suggestion
  expect(processedSchema.properties.test.anyOf).toStrictEqual([{ title: 'decimal', type: 'string', pattern: '^-?\\d+(\\.\\d{1,})?$' }])
```